src/constraints.jl

```diff
@@ -305,6 +305,22 @@ function is_valid(model::Model, con_ref::ConstraintRef{<:AbstractModel})
     )
 end
 
+"""
+    is_valid(model::Model,
+             c::ConstraintRef{Model,NonlinearConstraintIndex})
+
+Return `true` if `constraint_ref` refers to a valid nonlinear constraint in
+    `model`.
+"""
+function is_valid(model::Model,
+                  c::ConstraintRef{Model,NonlinearConstraintIndex})
```

Comment:
This is going to fail the format check. Wait for CI to fail and follow the diff, or use https://jump.dev/JuMP.jl/stable/developers/style/#JuliaFormatter.

The formatter doesn't format docstrings, but you should update it to match, and fix the indent on line 313.

Reply:
done